hack/istio/multicluster/kiali-prepare-remote-cluster.sh

```diff
@@ -484,6 +507,7 @@ info REMOTE_CLUSTER_CONTEXT=${REMOTE_CLUSTER_CONTEXT}
 info REMOTE_CLUSTER_NAME=${REMOTE_CLUSTER_NAME}
 info REMOTE_CLUSTER_NAMESPACE=${REMOTE_CLUSTER_NAMESPACE}
 info VIEW_ONLY=${VIEW_ONLY}
+info EXEC_AUTH_JSON=${EXEC_AUTH_JSON}
```

Comment:
Wrap in quotes, otherwise only the first line is logged.

```suggestion
info EXEC_AUTH_JSON="${EXEC_AUTH_JSON}"
```

Reply:
@frauniki don't do this if the "EXEC_AUTH_JSON" content is supposed to have real credentials. We don't want things like passwords or keys being printed here if they are sensitive. But if this exec auth section isn't anything more than setting up a cmd line and its arguments, then we should want to output it all, not just the first line.

Reply:
this fix is included in my [PR](https://github.com/frauniki/kiali/pull/1)